src/main/java/dev/mlml/command/impl/Help.java

```diff
@@ -13,16 +13,17 @@
         permissions = {Permission.MESSAGE_SEND}
 )
 public class Help extends Command {
-    public Help(Context ctx) {
-        super(new StringArgument.Builder("Command name")
-                      .description("The text to echo")
-                      .require()
+    public Help() {
+        super(new StringArgument.Builder("command")
+                      .description("Command to get details on")
                       .get()
         );
     }
 
     @Override
-    public void execute(Context args) {
-        args.getMessage().reply("Helptext placeholder").queue();
+    public void execute(Context ctx) {
+        String commandName = (String) ctx.getArgument("command").getValue();
+
+        System.out.println(commandName);
```

Comment:
Please use slf4j instead of System.out.println

Reply:
That should not have been there.